app/db/mongodb.py

```
@@ -129,6 +154,26 @@ def get_prod_mongo_client() -> MongoClient:
+                ssh_port=settings.prod_mongo_ssh_port,
+                ssh_user=settings.prod_mongo_ssh_user,
+                ssh_pkey=settings.prod_mongo_ssh_pem_key_path,
+                remote_bind_address=("127.0.0.1", port),
+                local_bind_port=settings.prod_mongo_local_bind_port,
+            )
+            # 터널을 통해 접속하므로 호스트와 포트 변경
+            host = "localhost"
+            port = settings.prod_mongo_local_bind_port
+            logger.info(f"Prod MongoDB SSH tunnel established on localhost:{port}")
+        except Exception as e:
+            logger.error(f"Failed to establish SSH tunnel for Prod MongoDB: {e}")
```

Ruff (0.14.7)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)